src/manager.rs

```diff
@@ -40,7 +40,7 @@ pub struct Manager {
 }
 
 impl Manager {
-    fn new() -> Manager {
+    pub fn new() -> Manager {
```

Comment:
I don’t mind allowing `Manager` to be created for tests, but in that case please do so via a `#[cfg(test)]` `pub(crate)` method called `new_for_test` or something.